src/wp-includes/load.php

```diff
@@ -948,6 +948,10 @@ function wp_get_active_and_valid_themes() {
 		return $themes;
 	}
 
+	if ( ! wp_using_themes() ) {
```

Comment:
@danielbachhuber Would `STYLESHEETPATH` be empty in this context?

Reply:
@spacedmonkey Unfortunately not. It's set to an erroneous value:

```diff
diff --git a/src/wp-includes/load.php b/src/wp-includes/load.php
index 9780519856..bc0a35ab0e 100644
--- a/src/wp-includes/load.php
+++ b/src/wp-includes/load.php
@@ -948,6 +948,9 @@ function wp_get_active_and_valid_themes() {
 		return $themes;
 	}
 
+	var_dump( 'TEMPLATEPATH: ' . TEMPLATEPATH );
+	var_dump( 'STYLESHEETPATH: ' . STYLESHEETPATH );
+
 	if ( ! wp_using_themes() ) {
 		return $themes;
 	}

```

```
$ wp --skip-themes option get home
string(82) "TEMPLATEPATH: /Users/danielbachhuber/projects/wordpress-develop/wp-content/themes/"
string(84) "STYLESHEETPATH: /Users/danielbachhuber/projects/wordpress-develop/wp-content/themes/"
https://wordpress-develop.test
```